python/turbodbc/cursor.py

```diff
@@ -289,11 +289,11 @@ def _numpy_batch_generator(self):
             first_run = False
             yield result_batch
 
-    def fetchallarrow(self):
+    def fetchallarrow(self, strings_as_dictionary=False):
```

Comment:
Could you please add a docstring for this function, so that it appears in the reference?